application/controllers/Login.php

```diff
@@ -59,19 +59,19 @@ public function validate()
             $username = request('username');
 
             if (empty($username)) {
-                throw new InvalidArgumentException('No username value provided.');
+                throw new InvalidArgumentException(lang('no_username_value_provided'));
             }
 
             $password = request('password');
 
             if (empty($password)) {
-                throw new InvalidArgumentException('No password value provided.');
+                throw new InvalidArgumentException(lang('no_password_value_provided'));
```

Comment:
This does not need to be translated because the JS will not allow for empty username submissions any longer

Reply:
Done, see https://github.com/alextselegidis/easyappointments/pull/1484/commits/3a0f41d2a9589312cc72c5578ec9ac7e9331ebad